test/common/http/conn_pool_grid_test.cc

```diff
@@ -110,11 +110,17 @@ class ConnectivityGridTest : public Event::TestUsingSimulatedTime, public testin
         quic_stat_names_(store_.symbolTable()) {}
 
   void initialize() {
+#ifdef ENVOY_ENABLE_QUIC
+    quic_connection_persistent_info_ =
+        std::make_unique<Quic::PersistentQuicInfoImpl>(dispatcher_, 0);
```

Comment:
again if we go the pure interface route, the implementation of the mock here can be empty, or have code.